### PR TITLE
Dashboard : Device value on 2 lines

### LINKED ISSUE #925: Dashboard : Device value on 2 lines
I don't know why...
Browser : Opera GX on Mac OS
Same thing on Chrome, obviously (Opera based on Chrome)

Capture d’écran 2020-11-08 à 14 28 44

### PR DESCRIPTION
**Pull request by atrovato**
Fixes #925

Do not allow to display device value on multiple lines (except for "no value received").

### CODE REVIEW

**Comment by Pierre-Gilles** [comment]
So how does it looks like when the text is long ?

File: front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx [hunk 1]
```diff
@@ -23,7 +24,7 @@ const SensorDeviceType = ({ children, ...props }) => (
     </td>
     <td>{props.deviceFeature.name}</td>
     {SPECIAL_SENSORS.indexOf(props.deviceFeature.category) === -1 && (
-      <td class="text-right">
+      <td class={cx('text-right', { 'text-nowrap': props.deviceFeature.last_value })}>
```
Comment (Pierre-Gilles) [inline front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx]: Can you do a stricter type check on this ? If last_value = 0, (ex: temperature = 0°C) then it'll be "false" here, but we want true. 

check only if null

File: front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx [hunk 2]
```diff
@@ -51,7 +52,7 @@ const SensorDeviceType = ({ children, ...props }) => (
       </td>
     )}
     {props.deviceFeature.category === DEVICE_FEATURE_CATEGORIES.MOTION_SENSOR && (
-      <td class="text-right">
+      <td class={cx('text-right', { 'text-nowrap': props.deviceFeature.last_value })}>
```
Comment (Pierre-Gilles) [inline front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx]: same here

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Thanks for the PR ! 

Just a little feedback on type, nothing serious but this can lead to very weird bugs

File: front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx (hunk 2 above)
Comment (atrovato) [reply front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx]: Here it should be `last_value_changed` not `last_value`.

**Comment by atrovato** [comment]
Before:



After:

**Comment by Pierre-Gilles** [comment]
Thanks for your changes! Good for me.

**Pull request merged**